CHANGES.md

```diff
@@ -30,6 +30,7 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 * Fixed a race condition in L4LB. (DCOS_OSS-4939)
 
 * `docker-gc` now removes unused volumes. (DCOS_OSS-1502)
+* Consolidated Telegraf for workloads that emit a large number of metrics. (DCOS-50994)
```

Comment:
Hmm, this shouldn't be here.

Reply:
Oh no - sorry good spot - something messed up with cherry-pick!